server/Game.js

```diff
@@ -154,21 +172,24 @@ export default class {
     }
   }
 
+  setAvailableWords() {
+    this.availableWords = [...words].filter(word => !this.words.includes(word));
+  }
+
   sendWordCandidates() {
+    const MAX_CANDIDATES = 3;
```

Comment:
Let's move this into `../shared/constants`, even though we don't share it yet, it's nice to have all constants in one place :-)